slate/locales/development.yml

```diff
@@ -20,3 +20,4 @@ development:
   biocacheServiceApiUrl: https://api.test.ala.org.au/biocachev2
   spatialServiceApiUrl: https://api.test.ala.org.au/spatial
   tokensAppUrl: https://tokens.dev.ala.org.au
+  ecodataApiUrl: https://api.test.ala.org.au/ecodata
```

Comment:
You can also add the `ecodataApiUrl` now to the staging and prod config files. e.g. `https://api.ala.org.au/ecodata`.